### PR TITLE
simple line to show you were offline

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers adds a little line on top of the right pane if you loaded messages but maybe didn't get them all

cc: @mmaxim

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by keybase-ci-visdiff** [comment]
The commits e4e3e93e5f94...b504840f79fe introduce visual changes on linux. <details><summary>:mag_right: 13 changed</summary>

 * Changed: **Revoke_Proof-https_website** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-https_website-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-https_website-b504840f79fe.png)  
   Revoke_Proof-https_website comparison
 * Changed: **Revoke_Proof-Zcash** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Zcash-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Zcash-b504840f79fe.png)  
   Revoke_Proof-Zcash comparison
 * Changed: **Revoke_Proof-Website** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Website-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Website-b504840f79fe.png)  
   Revoke_Proof-Website comparison
 * Changed: **Revoke_Proof-Twitter_Waiting** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Twitter_Waiting-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Twitter_Waiting-b504840f79fe.png)  
   Revoke_Proof-Twitter_Waiting comparison
 * Changed: **Revoke_Proof-Twitter_Error** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Twitter_Error-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Twitter_Error-b504840f79fe.png)  
   Revoke_Proof-Twitter_Error comparison
 * Changed: **Revoke_Proof-Twitter** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Twitter-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Twitter-b504840f79fe.png)  
   Revoke_Proof-Twitter comparison
 * Changed: **Revoke_Proof-Reddit** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Reddit-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Reddit-b504840f79fe.png)  
   Revoke_Proof-Reddit comparison
 * Changed: **Revoke_Proof-Hacker_News** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Hacker_News-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Hacker_News-b504840f79fe.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Hacker_News.png)
 * Changed: **Revoke_Proof-Git_Hub** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Git_Hub-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Git_Hub-b504840f79fe.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Git_Hub.png)
 * Changed: **Revoke_Proof-Facebook** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Facebook-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Facebook-b504840f79fe.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Facebook.png)
 * Changed: **Revoke_Proof-DNS** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-DNS-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-DNS-b504840f79fe.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-DNS.png)
 * Changed: **Revoke_Proof-Coinbase** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Coinbase-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Coinbase-b504840f79fe.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Coinbase.png)
 * Changed: **Revoke_Proof-Bitcoin** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Bitcoin-e4e3e93e5f94.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Bitcoin-b504840f79fe.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1490292076589-e4e3e93e5f94-b504840f79fe-linux/Revoke_Proof-Bitcoin.png)
</summary>

File: shared/constants/chat.js [hunk 1]
```diff
@@ -200,6 +201,7 @@ export type ConversationState = Record<{
   moreToLoad: boolean,
   isRequesting: boolean,
   isStale: boolean,
+  loadedOffline: boolean,
```
Comment (MarcoPolo) [inline shared/constants/chat.js]: Should we have this as a separate map from convID to loadedOffline?

Comment (chrisnojima) [inline shared/constants/chat.js]: this'll all change as it gets entitiy-ized

**Pull request merged**